File: go/Montecarlo_s.go
package main

import "fmt"
import "math/rand"
import "time"

func main() {
	count := 0.0
	lim := 2147483647
	seed := rand.NewSource(time.Now().UnixNano())
	random := rand.New(seed)
	for i := 0; i < lim; i++ {
		x := random.Float64()
		y := random.Float64()
		if x*x + y*y <= 1 {
			count++
		}
	}
	limf := 2147483647.0
	pi := (4.0 * (count*1.0)/limf)
	fmt.Println(pi)
}

File: python/Montecarlo_s.py
#!/usr/bin/python

import random, sys

val = 0
lim = 2147483647
i = 0
while i < lim:
	y = random.random()
	x = random.random()
	if x**2 + y ** 2 < 1:
		val += 1
	i += 1
print 4.0 * val/lim
